backend/models/todo-model.js

```diff
@@ -0,0 +1,6 @@
+import { string, boolean } from 'yup';
+
+export const todoFieldsSchema = {
+    text: string().required(),
+    is_checked: boolean().required()
+}
```

Comment:
You can wrap this straight with `object` from `yup` here, this schema should be static all the way through